### ISSUE TITLE
Blank screen when choosing collection type

### ISSUE THREAD

**Issue by UweSchuetzenmeister** [vivmagarwal/strapi-plugin-custom-api-builder#21]
When I try to add a collection type I get a blank screen and the console says 
`TypeError: S is not a function`

Strapi v4.11.3

Any idea?

Thanks for support

**Comment by emmismtv** [comment]
I'm having the same exact issue.

**Comment by jawadharaba98** [comment]
Same here

**Comment by ffdfo** [comment]
Same here

**Comment by Noou-Ben** [comment]
Me too :/

**Comment by aHalvarsson** [comment]
Strapi 4.13.6 and node 18.16.0
Same problem

**Comment by Aniruddh-J** [comment]
Yep, I thought it was just me. I am also getting the same error on Strapi 4.13.6.

**Comment by letops** [comment]
Same happens in Strapi 4.14.0
It only crashes on the client-side, no other problems are reported inside the terminal

**Comment by chestongo** [comment]
Blank screen when navigating from the Custom API route to another.

**Comment by CurlyTaleGames** [comment]
Same problem on Strapi 4.15.5

**Comment by rahul-ti** [comment]
did anyone figure out this issue?

**Comment by rahul-ti** [comment]
@vivmagarwal please help

**Comment by vivmagarwal** [comment]
:( sorry for the delay! I have made and tested several changes to sync the plugin with the API changes in Strapi. Please see if it works now!

![image](https://github.com/user-attachments/assets/2a916531-08f2-4810-9f8d-7ff8249da913)
![image](https://github.com/user-attachments/assets/42a053ae-3f1c-4e15-99ec-fcca5dbce90b)
